_i18n/nb-no/resources/user-guides/multisig-messaging-system.md

```diff
@@ -0,0 +1,864 @@
+    list        Lister alle meldinger
+    next        Evaluerer de(n) neste mulige multisig-handlingen(e) i henhold til lommebokens status, eksekvererbarhet og valgtilbud 
+    sync        Tvinger generering av synkroniseringsdata for multisig, uavhengig av lommebokstatus, for gjenoppretting fra spesielle situasjoner som «foreldet data»-feilmeldinger
+    transfer    Initierer overføring med MMS-støtte; argumenter er identiske til normale «transfer»-kommandoer. For mer informasjon, du se mer der
+    delete      Sletter en enkel melding ved å oppgi ID-en, eller sletter alle meldinger ved å bruke 'all'
+    send        Sender en enkel melding ved å oppgi ID-en, eller sender alle ventende meldinger
+    receive     Sjekker umiddelbart etter nye, innkommende meldinger
+    note        Sender et beskjed på én linje til en signerer, identifisert av merkelappen, eller viser alle uleste beskjeder
+    show        Viser detaljert informasjon om en enkel melding
+    export      Eksporterer innholdet av en melding til en fil
+    set         Angir valg, der 'auto-send' så langt er den eneste
+
+    start_auto_config   Starter auto-config-prosessen på auto-config-eierens lommebok ved å opprette nye tokens
+    auto_config         Starter auto-config ved å bruke tokenen som ble mottatt av auto-config-eieren
+    stop_auto_config    Sletter eventuelle tokens og avslutter en auto-config-prosess
+    send_signer_config  Sender din komplette signererkonfigurasjon til alle andre signerere
+    
+Du får listen over kommandoer ved å eksekvere `help mms`, og hjelp til en bestemt underkommando ved å bruke `help mms <underkommando>`, f.eks. `help mms next`. Du kan alternativt bruke `mms help <underkommando>` hvis den føles mer naturlig.
+
+## Konfigurering av lommebok for bruk med MMS
+
+### Adresser og merkelapper
+
+For bedre forståelse av noen grunnleggende fakta om å adressere og henvise til signerere (eller lommebøkene deres) i MMS-en:
+
+Hvis du oppretter en ny lommebok, blir den (så klart) en egen en, med en unik Monero-adresse. Hvis du senere konfigurerer lommeboken for multisig, **endrer** lommeboken sin offentlige adresse til den vanlige multisig-adressen som du deler med alle de andre autoriserte signererne.
+
+MMS-en bruker den første, «originale» offentlige Monero-adressen over hele livsløpet til lommeboken for adressering før **og** etter den «går over til» multisig. Det kan være litt forvirrende at en lommebok bør ha **to** offentlige adresser, men når du har fått den opprinnelige adressen i signererkonfigurasjonen din, kan du mer eller mindre glemme den.
+
+MMS-en bruker *merkelapper* som lar deg navngi deg selv og andre signerere, og at MMS-kommandoen brukes når det referreres til signerere. (Å bruke Monero-adresser eller Bitmessage-adresser i slike kommandoer hadde vært ganske så ubeleilig.)

```

Comment:
Change to: MMS-en bruker *merkelapper* som lar deg navngi deg selv og andre signerere, og MMS-kommandoen brukes når det refereres til signerere.

Reply:
fixed